src/foam/core/notification/push/APNSPushService.js

```diff
@@ -33,6 +34,18 @@ foam.CLASS({
       name: 'TTL_IN_HOURS',
       documentation: 'Time to live for the notification in hours',
       value: 12
+    },
+    {
+      type: 'Long',
```

Comment:
Why are these constants and not properties? I expect them to be modified/tweaked at runtime.

Reply:
updated